Upgrading Suru++ Asprómauros

gusbemacbe opened the pull request:
Hello @actionless !

I have just upgraded Suru++ Asprómauros, adding gradients support, including @Roboron3042's Cyberpunk Neon, for **all** the icons of `actions`, `devices`, `emblems`, `emotes`, `panel` and `status`, with their `symbolic` folders. 

Review the files I have just updated and see if they are all OK. If you find wrong, tell me where I have to fix.

actionless commented:
is the script working for you after that change?

https://travis-ci.com/themix-project/oomox/jobs/205589435

actionless commented on plugins/icons_suruplus_aspromauros/change_color.sh:
```diff
@@ -136,21 +139,27 @@ fi
 		-type f -name '*.svg' \
 		-exec sed -i'' \
-			-e 's/currentColor/url(#oomox)/g' \
+		# For avoiding to overwrite `fill|stroke="currentColor" class="ColorScheme-Highlight"`
```
you can't put comments between `\` escaped splitlnes

gusbemacbe replied:
Done

gusbemacbe commented:
Hello @actionless!

It used to work before making a comment. When added a comment, I was not aware that the added comment caused Oomox to break. Sorry, I have just fixed it and it worked without the comment.

I added that comment because I wanted to explain you why those long replacements.

actionless commented:
thanks a lot!
